go/kbfs/libkbfs/favorites.go

```diff
@@ -300,6 +300,56 @@ func (f *Favorites) closeReq(req *favReq, err error) {
 	}
 }
 
+func (f *Favorites) crossCheckWithEditHistory() {
+	// NOTE: Ideally we would wait until all edit activity processing
+	// had completed in the FBO before we do these checks, but I think
+	// in practice when the mtime changes, it'll be the edit activity
+	// processing that actually kicks off these favorites activity,
+	// and so that particular race won't be an issue.  If we see
+	// weirdness here though, it might be worth revisiting that
+	// assumption.
+
+	// Fix the favorite times to be the latest known edit history
+	// times, if possible.  If not possible, set them to be lower than
+	// the minimum known time, if they're not already.
```

Comment:
Can you mention in this comment the reason the times might be wrong (i.e. GC, etc.)?